modules/noise/noise.cpp

```diff
@@ -54,7 +54,11 @@ Vector<Ref<Image>> Noise::_get_seamless_image(int p_width, int p_height, int p_d
 
 Ref<Image> Noise::get_seamless_image(int p_width, int p_height, bool p_invert, bool p_in_3d_space, real_t p_blend_skirt, bool p_normalize) const {
 	Vector<Ref<Image>> images = _get_seamless_image(p_width, p_height, 1, p_invert, p_in_3d_space, p_blend_skirt, p_normalize);
-	return images[0];
+	if (images.size() == 0) {
+		return Ref<Image>();
+	} else {
+		return images[0];
+	}
```

Comment:
We usually do early returns to handle errors, then continue the non-failure case in the main scope.
```suggestion
	if (images.size() == 0) {
		return Ref<Image>();
	}
	return images[0];
```

Reply:
Thanks!